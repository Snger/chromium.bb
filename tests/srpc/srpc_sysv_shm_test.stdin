shmem desc_shmem desc_address 65536
strcpy ${desc_address} 0 "Hello SysV shared memory."
rpc map_sysv_handle h(desc_shmem) *
