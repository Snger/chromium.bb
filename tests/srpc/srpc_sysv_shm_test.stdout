rpc call intiated map_sysv_handle:h:
input 0:  h(desc_shmem)
string 'Hello SysV shared memory.'
rpc call complete map_sysv_handle:h:

